$OpenBSD$

archivers/minizip is too new to be used, force using the older bundle
https://github.com/desktop-app/cmake_helpers/pull/156

Fixed upstream, remove this patch upon the next update.

Index: cmake/external/minizip/CMakeLists.txt
--- cmake/external/minizip/CMakeLists.txt.orig
+++ cmake/external/minizip/CMakeLists.txt
@@ -9,7 +9,7 @@ add_library(desktop-app::external_minizip ALIAS extern
 
 if (DESKTOP_APP_USE_PACKAGED)
     find_package(PkgConfig REQUIRED)
-    pkg_check_modules(MINIZIP IMPORTED_TARGET minizip)
+    pkg_check_modules(MINIZIP IMPORTED_TARGET minizip<2.0.0)
 
     if (MINIZIP_FOUND)
         target_link_libraries(external_minizip INTERFACE PkgConfig::MINIZIP)
